docs/api/components/schemas.yaml

```diff
@@ -1165,7 +1165,9 @@ Database:
       description: The name of the database.
       type: string
     sync:
-      description: The Javascript function that newly created documents are ran through for the _default scope and collection.
+      description: The Javascript function that newly created documents are ran through for the default scope and collection.
+
+        If `scopes` parameter is set, this is ignored.
```

Comment:
Should we add SG config validation to cover this case - to explicitly disallow it?

Reply:
Already done in https://github.com/couchbase/sync_gateway/blob/ae024a247c0ee62a5598121843e9904084ac15e1/rest/config.go#L852